src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String CSV_SEPARATOR = ",";
+    public static final int ERROR_EXIT_CODE = -1;
+    public static final int CSV_OP_TYPE_INDEX = 0;
+    public static final int CSV_VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileContent = readFromFile(fromFileName);
+        String report = getReport(fileContent);
+        writeToFile(toFileName, report);
+    }
+
+    private void writeToFile(String toFileName, String report) {
```

Comment:
it is better to place private methods in the call sequence (just a recommendation)